### ISSUE TITLE
possible test failure in upcoming Julia version 1.5 

### ISSUE THREAD

**Issue by rfourquet** [chrisvwx/LLLplus.jl#39]
A PkgEval run for a Julia [pull request](https://github.com/JuliaLang/julia/pull/29240) which changes the generated numbers for `rand(a:b)` indicates that the tests of this package might fail in Julia 1.5 (and on Julia current master branch).

Also, you might be interested in using the new [StableRNGs.jl](https://github.com/rfourquet/StableRNGs.jl) registered package, which provides guaranteed stable streams of random numbers across Julia releases.

Apologies if this is a false positive. Cf. https://github.com/JuliaCI/NanosoldierReports/blob/ab6676206b210325500b4f4619fa711f2d7429d2/pkgeval/by_hash/52c2272_vs_47c55db/logs/LLLplus/1.5.0-DEV-87d2a04de3.log

**Comment by chrisvwx** [comment]
Thanks!  I will look at this more next week; I may wait until we're closer to 1.5 to address it.

 StableRNGs looks like it's your package; it looks interesting and useful.  If I read the README right, you're using a linear congruential generator (LCG). As a bit of trivia, I think the LLL can be used to implement Knuth's Spectral test of LCGs; see [section 3.3.4 of vol 2](https://github.com/djtrack16/thyme/blob/master/computer%20science/Donald.E.Knuth.The.Art.of.Computer.Programming.Volume.2.pdf)

**Comment by rfourquet** [comment]
> StableRNGs looks like it's your package

Yes I made it because some people dislike that the RNG can change in minor Julia version :)
Thank you for the pointer to Knuth's Spectral test, I will have a closer look when I have time. I would believe BigCrush includes such a test, but I will have to check.
